Add authorization on the scim/Users/{id} endpoint

garaimanoj opened the pull request.

federicaagostini committed:
Fix scope policy filter when refresh token has been already issued

garaimanoj committed:
Add authorization on the scim/Users/{id} endpoint

garaimanoj commented:
@rmiccoli :The authZ on the GET endpoint is ok, but I am not sure about the PATCH, as it is now possible for a user to change their username, status, etc. It differs from what is is allowed with the PATCH on the scim/Me endpoint.

enricovianello committed:
Fix Sonar issues

rmiccoli committed:
Integrate MFA with external OIDC and SAML providers (#918)

enricovianello committed:
Update CHANGELOG.md

enricovianello committed:
Add v1.12.0 info on CHANGELOG

enricovianello committed:
Fix text alignment for btn-social buttons

enricovianello committed:
Fix Eclipse missing support of surefire argLine variable

enricovianello committed:
Back to voms-api-java v3.3.4

garaimanoj committed:
Add check for admin and user's patch operation

garaimanoj committed:
Remove duplicate code

garaimanoj committed:
Change private acces

garaimanoj committed:
Fix sonar issue

rmiccoli reviewed (approved):
This PR looks good to me :)

enricovianello committed:
Bump to version 1.13.0

S203932 committed:
Include Subject DN field when searching for a user (#952)

garaimanoj committed:
Store user affiliation into user info table (#781)

* Display affiliation in user profile

federicaagostini committed:
Return only FQANs with parent group equal to the VO name in the proxy (#993)

garaimanoj committed:
Add endpoint to get the list of clients owned by a user (#974)

mmutadich committed:
Option to limit custom scopes to admins only (#994)

rmiccoli committed:
Allow users to remove themselves from groups (#960)

mmutadich committed:
SCIM API bulk operations (#969)

mmutadich committed:
Configurable support button (#998)

federicaagostini committed:
Unlink certificate using both subject and issuer DNs (#1008)

Added issuer to the related AUDIT event
Delete duplicated and unused event class

garaimanoj committed:
Remove EditableFields

garaimanoj committed:
Fix test

garaimanoj committed:
Fix test

garaimanoj commented:
Due to merge conflict, replaced by [PR](https://github.com/indigo-iam/iam/pull/1010)
